website/docs/r/log_search.html.markdown

```diff
@@ -73,7 +73,17 @@ The following arguments are supported:
 - `description` - (Optional) Description of the search.
 - `parent_id` - (Required) The identifier of the folder to create the log search in.
 - `query_string` - (Required) Log query to perform.
-- `query_parameters` - (Optional) TODO Find a good description.
+- `query_parameters` - (Optional) Array of objects (LogSearchQueryParameterSyncDefinitionBase). Values for search template used in the search query. 
+    Learn more about the search templates here : https://help.sumologic.com/docs/search/get-started-with-search/build-search/search-templates/
+    Schema for `LogSearchQueryParameterSyncDefinitionBase`
+        - `name` - (Required) The name of the parameter.
+        - `description` - (Optional) A description of the parameter.
+        - `dataType` - (Required) The data type of the parameter. Supported values are:
+            1. `NUMBER`
+            2. `STRING`
+            3. `ANY`
+            4. `KEYWORD`
+        - `value` - A value for the parameter. Should be compatible with the type set in dataType field.
```

Comment:
I think you should added `(Required)` here too, since you did that for the other arguments.